pkg/reader/ccip.go

```diff
@@ -550,6 +551,63 @@ func (r *ccipChainReader) GetChainFeePriceUpdate(ctx context.Context, selectors
 	return feeUpdates
 }
 
+func (r *ccipChainReader) GetRMNRemoteConfig(
+	ctx context.Context,
+	destChainSelector cciptypes.ChainSelector,
+) (rmntypes.RemoteConfig, error) {
+	if err := validateExtendedReaderExistence(r.contractReaders, destChainSelector); err != nil {
+		return rmntypes.RemoteConfig{}, err
+	}
+
+	contractAddress, err := r.GetContractAddress(consts.ContractNameRMNRemote, destChainSelector)
+	if err != nil {
+		return rmntypes.RemoteConfig{}, fmt.Errorf("get RMNRemote contract address: %w", err)
+	}
+
+	var vc versionnedConfig
+	err = r.contractReaders[destChainSelector].ExtendedGetLatestValue(
+		ctx,
+		consts.ContractNameRMNRemote,
+		consts.MethodNameGetVersionedConfig,
+		primitives.Unconfirmed,
+		map[string]any{},
+		&vc,
+	)
+	if err != nil {
+		return rmntypes.RemoteConfig{}, fmt.Errorf("get RMNRemote config: %w", err)
+	}
+
+	var dh string
```

Comment:
Hm I'm not sure this'll work, the contract returns `bytes32`, which means `cciptypes.Bytes32` or `[32]byte` depending on your needs (i.e JSON serialization)